SimTransport/PPSProtonTransport/BuildFile.xml

```diff
@@ -0,0 +1,15 @@
+<use   name="FWCore/Framework"/>
+<use   name="FWCore/ParameterSet"/>
+<use   name="CommonTools/Utils"/>
+<use   name="SimDataFormats/GeneratorProducts"/>
+<use   name="SimGeneral/HepPDTRecord"/>
+<use   name="PPSTools/Utilities"/>
+<use   name="SimTransport/TotemRPProtonTransportParametrization"/>
+<use   name="hector"/>
+<use   name="hepmc"/>
+<use   name="clhep"/>
+<use   name="root"/>
```

Comment:
please use the part of root you need

Reply:
done... using name="rootcore" (hope this is the correct way).